src/Config/Configuration.php

```diff
@@ -203,6 +208,28 @@ public function addPathToExclude(string $path): self
         return $this;
     }
 
+    /**
+     * @param list<string> $regexps
+     * @return $this
+     */
+    public function addRegexpsToExclude(array $regexps): self
+    {
+        foreach ($regexps as $regexp) {
+            $this->addRegexpToExclude($regexp);
+        }
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function addRegexpToExclude(string $regexp): self
```

Comment:
I believe the given regex should be validated at this point similary to what `ignoreUnknownClassesRegex` does.

Reply:
https://github.com/shipmonk-rnd/composer-dependency-analyser/pull/140/commits/7cdb4b920ca61419c790a49158126d96ab0b0739

Reply:
https://github.com/shipmonk-rnd/composer-dependency-analyser/pull/140/commits/65b12bbf6a36bceed90cc4e589f9a98e0b587c67